aztec_gddt/params.py

```diff
@@ -296,9 +291,14 @@ def steady_state_blob_gas_estimate(state: AztecModelState):
     gas_fee_blob_time_series=GAS_FEE_BLOB_TIME_SERIES_LIST[-1],
     commit_bond_amount=16.0,  # type: Tokens
     op_cost_sequencer=0,  # Assumption: Currently all Sequencers have one op_cost constant to evaluate against
-    op_cost_prover=0, # Assumption: Currently all Provers have one op_cost constant to evaluate against
-    safety_factor_commit_bond = 0.0,
-    safety_factor_reveal_content = 0.0,
-    safety_factor_rollup_proof = 0.0,
-    past_gas_weight_fraction = 0.9
+    op_cost_prover=0,  # Assumption: Currently all Provers have one op_cost constant to evaluate against
+    safety_factor_commit_bond=0.0,
+    safety_factor_reveal_content=0.0,
+    safety_factor_rollup_proof=0.0,
+    past_gas_weight_fraction=0.9,
```

Comment:
Make sure to add the new parameters into the `AztecModelParams` definition at `types.py`